mappings/net/minecraft/util/math/Vec3i.mapping

```diff
@@ -3,9 +3,26 @@ CLASS fq net/minecraft/util/math/Vec3i
 	FIELD b y I
 	FIELD c z I
 	FIELD e ZERO Lfq;
-	METHOD a squaredDistanceToCenter (DDDZ)D
+	METHOD a getDistanceSq (DDDZ)D
+		ARG 1 x
+		ARG 3 y
+		ARG 5 z
+		ARG 7 treatAsBlockPos
+	METHOD a isWithinDistance (Lfk;D)Z
+		ARG 1 pos
+		ARG 2 distance
+	METHOD a getSquaredDistance (Lfk;Z)D
+		ARG 1 pos
+		ARG 2 treatAsBlockPos
+	METHOD a isWithinDistance (Lfq;D)Z
+		ARG 1 vec
+		ARG 2 distance
 	METHOD d crossProduct (Lfq;)Lfq;
-	METHOD m squaredDistanceTo (Lfq;)D
+		ARG 1 vec
+	METHOD m getSquaredDistance (Lfq;)D
+		ARG 1 vec
+	METHOD n getTaxicabDistance (Lfq;)I
```

Comment:
Is taxicab or manhattan distance better known?

Reply:
I've only really heard Manhattan be used, imo that is more clear

Reply:
Changed it